tests/plugins/release-monitor/index.js

```diff
@@ -0,0 +1,726 @@
+                payload: {
+                    action: "closed",
+                    installation: {
+                        id: 1
+                    },
+                    issue: {
+                        labels: [
+                            {
+                                name: "test"
+                            }
+                        ],
+                        number: 5
+                    },
+                    repository: {
+                        name: "repo-test",
+                        owner: {
+                            login: "test"
+                        }
+                    }
+                }
+            });
+
+            expect(pr1Status.isDone()).toBeFalsy();
+            expect(pr2Status.isDone()).toBeFalsy();
+        });
+    });
+
+    ["opened", "reopened", "synchronize", "edited"].forEach(action => {
+        describe(`pull request ${action}`, () => {
+            test("put pending for non semver patch PR under post release phase", async() => {
+                mockAllOpenPrWithCommits([
+                    {
+                        number: 1,
+                        title: "New: add 1",
+                        commits: [
+                            {
+                                sha: "111old",
+                                commit: {
+                                    message: "New: add 1"
+                                }
+                            },
+                            {
+                                sha: "111",
+                                commit: {
+                                    message: "New: add 1"
+                                }
+                            }
+                        ]
+                    }
+                ]);
+
+                nock("https://api.github.com")
+                    .get("/repos/test/repo-test/issues?labels=release%2Cpost-release")
+                    .reply(200, [{}]);
+
+                const newPrStatus = nock("https://api.github.com")
+                    .post("/repos/test/repo-test/statuses/111")
+                    .reply(200, assertPendingStatus);
+
+                await bot.receive({
+                    event: "pull_request",
+                    payload: {
+                        action: "opened",
+                        installation: {
+                            id: 1
+                        },
+                        pull_request: {
+                            number: 1,
+                            title: "New: add 1"
+                        },
+                        repository: {
+                            name: "repo-test",
+                            owner: {
+                                login: "test"
+                            }
+                        }
+                    }
+                });
+
+                expect(newPrStatus.isDone()).toBeTruthy();
+            });
+
+            test("put success for non semver patch PR under post release phase", async() => {
```

Comment:
Not fully sure I understand this test. Is this test for outside of a release cycle (i.e., no open "release" issue with or without "post-release")? Also, the mock commit below looks like it should be semver-patch.